README.rst

```diff
@@ -482,7 +482,7 @@ Advanced Concepts
 Logging
 .......
 
-To use the logger, use the standard library ``logging`` module::
+To use the logger, use the standard SDK ``logging`` module::
```

Comment:
```suggestion
To use the logger, use the standard library ``logging`` module::
```
should stay "standard library" imo

Reply:
ah, didn't see that it was called library before!